ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -138,7 +138,7 @@ func (r *RayServiceReconciler) Reconcile(ctx context.Context, request ctrl.Reque
 		}
 	} else if activeRayClusterInstance != nil && pendingRayClusterInstance != nil {
 		if err = r.updateStatusForActiveCluster(ctx, rayServiceInstance, activeRayClusterInstance, logger); err != nil {
-			logger.Error(err, "The updating of the status for active ray cluster while we have pending cluster failed")
+			logger.Error(err, "Active Ray cluster's status update failed.")
```

Comment:
"Update active ray cluster's status failed." may be better

Reply:
I changed it to `"Failed to update active Ray cluster's status."`